packages/flutter_tools/test/integration.shard/debug_adapter/flutter_adapter_test.dart

```diff
@@ -34,236 +36,324 @@ void main() {
     tryToDelete(tempDir);
   });
 
-  testWithoutContext('can run and terminate a Flutter app in debug mode', () async {
```

Comment:
this file may be best reviewed ignoring whitespace or side-by-side, as I wrapped a group around the `launch` tests that makes it seem like almost the entire file was modified.

Reply:
:+1: